ThunkLibs/GuestLibs/CMakeLists.txt

```diff
@@ -41,8 +42,10 @@ function(generate NAME SOURCE_FILE)
       DEPENDS "${GENERATOR_EXE}"
       DEPENDS "${SOURCE_FILE}"
       COMMAND "${GENERATOR_EXE}" "${SOURCE_FILE}" "${NAME}" "-${WHAT}" "${OUTFILE}" -- -std=c++17 -DGUEST_THUNK_LIBRARY
-            # Expand include directories to space-separated list of -isystem parameters
-           "$<$<BOOL:${prop}>:;-isystem$<JOIN:${prop},;-isystem>>"
+        # Expand compile definitions to space-separated list of -D parameters
```

Comment:
Why is this needed?

Reply:
compile definitions automatically get stripped of their `-D` in cmake, so we need to add them back in to use this interface.